src/TritonRoute/src/pa/FlexPA_prep.cpp

```diff
@@ -853,49 +854,42 @@ void FlexPA::prepPoint_pin_checkPoint_via(
     auto via = make_unique<frVia>(viaDef);
     via->setOrigin(bp);
     via->getLayer1BBox(box);
+    if (instTerm) {
```

Comment:
Why are we checking if instTerm exists if none of the data below access it? Shouldn't we be checking for the value we'd be interested in instead?

Reply:
instTerm is being checked because it may or not be nullptr. If it is not, then I need to do something.